src/components/basic_table/in_memory_table.test.tsx

```diff
@@ -924,7 +928,8 @@ describe('EuiInMemoryTable', () => {
 
       expect(props.onTableChange).toHaveBeenCalledTimes(0);
       component
-        .find('EuiPaginationButton[data-test-subj="pagination-button-1"]')
+        .find('[data-test-subj="pagination-button-1"]')
```

Comment:
Let's change to

```suggestion
        .find('EuiButtonEmpty[data-test-subj="pagination-button-1"]')
```

and drop the `.first()`. Seeing `first` in this chain implied (to me) that having multiple buttons with that test subject is valid, when it's not true - there is only one, but the test environment finds the virtual dom node as well. Changing to explicitly look for `EuiButtonEmpty` helps narrow the test as needed but will also catch a future regression if two of these end up rendering.